src/components/Assistant.vue

```diff
@@ -14,34 +15,45 @@
 				<template #icon>
 					<CreationIcon :size="20" class="icon" />
 				</template>
-				<NcActionButton v-for="type in taskTypes"
+				<NcActionButton
+					v-for="type in taskTypes"
```

Comment:
I think pretty much everywhere else in nextcloud we have the first attribute on the line with the tag name just like this was before.

I actually like the separation - but I care about consistency more. So not sure what best to do here.